python_modules/dagster/dagster/_record/__init__.py

```diff
@@ -57,23 +60,34 @@ def _get_field_set_and_defaults(
                     f"Conflicting non-abstract @property for field {name} on record {cls.__name__}."
                     "Add the the @abstractmethod decorator to make it abstract.",
                 )
-            elif isinstance(attr_val, _tuple_getter_type):
-                # When doing record inheritance, filter out tuplegetters from parents.
-                # This workaround only seems needed for py3.8
-                continue
-            else:
+            # When doing record inheritance, filter out tuplegetters from parents.
+            # This workaround only seems needed for py3.8
```

Comment:
if this is only needed for py3.8 can we ditch this? (we no longer support py38)

doesn't need to be this pr tho